aspnet/migration/rc1-to-rc2.rst

```diff
@@ -208,6 +232,21 @@ To compile views, set the ``preserveCompilationContext`` option in ``project.jso
 
 You no longer need to reference the Tag Helper package ``Microsoft.AspNet.Mvc.TagHelpers``, which was renamed to ``Microsoft.AspNetCore.Mvc.TagHelpers`` in RC2. The package is now referenced by MVC by default.
 
+
+Global.json
```

Comment:
@DamianEdwards I believe this section should be moved to the top? It's the single change that is going to change your RC1 app to RC2.